extensions/amp-story/1.0/amp-story.css

```diff
@@ -159,25 +159,25 @@ amp-story[standalone].amp-notbuilt {
 }
 
 amp-story[standalone]:-webkit-full-screen {
-  height: var(--story-100dvh) !important;
+  height: 100vh !important;
```

Comment:
Why isn't this using `var(--story-100dvh)`?

Reply:
Full screen pages don't have any issues with `vh` since the browser appbar is not showing, it should be always the same `1dvh = 1vh`.